nginx/ng/servers_config.sls

```diff
@@ -112,6 +112,11 @@ nginx_server_available_dir:
     - name: {{ server_curpath(server) }}
     - source: {{ source_path }}
     - template: jinja
+    - require:
+      - file: nginx_server_enabled_dir
+      - file: nginx_server_available_dir
```

Comment:
Note that this state id is wrapped in a conditional so it might not exist.

Reply:
Sorry, can you explain it a little bit further? Thanks!

Reply:
highstate might fail because the state id might not exist if `available_dir == enabled_dir`

Reply:
@EvaSDK thanks for checking this, effectively this is not needed. Going to push a commit to remove this. Thanks!